app/consumers/courses_consumer.rb

```diff
@@ -0,0 +1,14 @@
+require 'karafka'
+require_relative 'application_consumer'
+require 'plezi'
+
+class CourseConsumer < ApplicationConsumer
+  def consume
+	if ( Object.const_defined?('EventStream') == false )
+	  puts "EventStream class not initialized" 
+	else 
+	  EventStream.on_message(params)
```

Comment:
You cannot call `on_message`, it is called internally by plezi only when the websocket server receives a message from the client.